test/connectors/CustomsDeclareExportsMovementsConnectorSpec.scala

```diff
@@ -44,39 +51,91 @@ class CustomsDeclareExportsMovementsConnectorSpec extends ConnectorSpec {
 
       val request =
         MovementRequest("eori", "provider-id", MovementType.Arrival, ConsignmentReferences("ref", "value"), MovementDetailsRequest("datetime"))
-      await(connector.submit(request))
+      connector.submit(request).futureValue
```

Comment:
Why?

Reply:
In other services I've only seen `.futureValue`, so consistencyis the first reason. And I've not seen any difference in behavior. Therefore, if there is no functionality differences I'd like to be consistent.

Reply:
Theres a good split in the codebase of both styles, I don't think it matters really